### PR TITLE
fix(ingestion/lookml): enable stack-trace in lookml logs

### PR DESCRIPTION
**Pull request by sid-acryl**
## Summary by CodeRabbit

- **New Features**
	- Enhanced error handling and logging for file loading and parsing processes, providing clearer error messages and file paths.

- **Bug Fixes**
	- Improved the clarity of failure messages for easier diagnostics of issues related to file loading and parsing.

- **Chores**
	- Added debug logging of raw file content upon parsing failure to assist in troubleshooting.

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py [hunk 1]
```diff
@@ -60,7 +60,12 @@ def _load_viewfile(
             with open(path) as file:
                 raw_file_content = file.read()
         except Exception as e:
-            self.reporter.failure("Failed to read lkml file", path, exc=e)
+            self.reporter.failure(
+                title="LKML File Loading Error",
+                message=f"Failed to read lkml file {path}",
+                exc=e,
+                log=True,
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py:67]: log=True is the default - no need to specific it explicitly

**Review by hsheth2 — APPROVED** [review]

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py (hunk 1 above)
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py:67]:
```suggestion
```

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py [hunk 2]
```diff
@@ -60,7 +60,12 @@ def _load_viewfile(
             with open(path) as file:
                 raw_file_content = file.read()
         except Exception as e:
-            self.reporter.failure("Failed to read lkml file", path, exc=e)
+            self.reporter.failure(
+                title="LKML File Loading Error",
+                message=f"Failed to read lkml file {path}",
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py:65]: the path should be in `context`, not `message`. Message and title should always be constant strings

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py [hunk 3]
```diff
@@ -86,7 +91,17 @@ def _load_viewfile(
...
+            self.reporter.failure(
+                title="LKML File Parsing Error",
+                message=f"Failed to parse lookml file {path}",
+                exc=e,
+                log=True,
+            )
+
+            logger.debug(f"Raw file content for path {path}")
+
+            logger.debug(raw_file_content)
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py:103]: make this one debug log message

**Review by hsheth2 — CHANGES_REQUESTED** [review]

File: metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py (hunk 3 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py:103]: They are already debug.

Comment (hsheth2) [reply metadata-ingestion/src/datahub/ingestion/source/looker/looker_file_loader.py:103]: src/datahub/ingestion/source/dbt/dbt_cloud.py

**Review by hsheth2 — APPROVED** [review]

**Pull request merged**